employees.html

```diff
@@ -0,0 +1,549 @@
+    <title>Employees-dashboard</title>
+  </head>
+  <body>
+    <div class="container">
+      <aside class="sidebar">
+        <section class="dashboard">
+          <figure>
+            <img src="images/tezo-logo.svg" alt="logo" class="logo" />
+            <img src="images/handle.png" alt="handle" class="sidebar-handle" />
+          </figure>
+          <div class="all-dashboard">
+            <div class="all-dashboard-heading">
+              <hr class="all-dashboard-heading-line" />
+              <p>ALL</p>
+            </div>
+            <ul>
+              <li>
+                <img src="images\horizontal-nav\Dashboard.svg" alt="" />
+                <a href="#"> Dashboard</a>
+              </li>
+
+              <li class="selected-menu">
+                <div class="all-dashboard-selected-menu">
+                  <img
+                    src="images\horizontal-nav\Employees.svg"
+                    alt="employees"
+                  />
+                  <a href="#"> Employees</a>
+                </div>
+                <div class="dashboard-selection-highlight"></div>
+              </li>
+            </ul>
+          </div>
+          <div class="role-management-dashboard">
+            <div class="role-management-dashboard-heading">
+              <hr class="role-management-dashboard-heading-line" />
+              <p>ROLE / USER MANAGEMENT</p>
+            </div>
+            <ul>
+              <li class="roles-selection-menu">
+                <div>
+                  <img
+                    src="images/horizontal-nav/Roles.svg"
+                    alt="Access-rights"
+                  />
+                  <a href="roles.html">Roles</a>
+                </div>
+                <img
+                  src="images/drop-right-icon.png"
+                  alt="drop-right"
+                  class="roles-selection-menu-icon"
+                />
+              </li>
+              <li class="roles-selection-menu">
+                <div>
+                  <img
+                    src="images\horizontal-nav\assign user.svg"
+                    alt="Access-rights"
+                  />
+                  <a href="#">Access Rights</a>
+                </div>
+                <img
+                  src="images/drop-right-icon.png"
+                  alt="drop-right"
+                  class="roles-selection-menu-icon"
+                />
+              </li>
+            </ul>
+          </div>
+        </section>
+        <div class="update">
+          <p class="update-install">Install new update 2.3.1A</p>
+          <p class="update-description">
+            Allow to launch the new update for the AD application
+          </p>
+          <div>
+            <button class="btn btn-dismiss"><a href="#">Dismiss</a></button
+            ><button class="btn btn-accept"><a href="#">Accept</a></button>
+          </div>
+        </div>
+      </aside>
+      <main class="main">
+        <section class="employees-navbar">

```

Comment:
In main header, as there are no nav links, u shouldn't name classes with navbar.
update the class names.

Reply:
main is just containing container navbar section is another independent section inside the conatning container

Reply:
u shouldnt use navbar name as there are no navigation links